docs/topics/proc_interacting-with-the-secured-booster-on-the-command-line.adoc

```diff
@@ -40,11 +40,7 @@ $ curl -sk -X POST https://<SSO_AUTH_SERVER_URL>/auth/realms/$REALM/protocol/ope
   -d client_secret=$SECRET
 ----
 +
-NOTE: The `-sk` option tells curl to ignore failures resulting from self-signed certificates. Do not use this option in a production environment.
-+
-On macOS, you must have `curl` version `7.56.1` or greater installed. It must also be built with OpenSSL.
-
-** Extract the access token information, for example using the `jq` tool:
+Optionally, you can extract the access token information, for example using the `jq` tool:
```

Comment:
I don't think that extracting the token is optional. You have to do that in order to invoke the service. What can be considered optional is the usage of the `jq` tool.

Reply:
But what RHOARDOC-1197 actually seems to ask for is a _single_ command that contains both `curl ... | jq ...`. I would personally agree that it's much better than splitting the command to two, but I guess you have split it for a reason (educative probably?).